docs/user-guide/README.md

```diff
@@ -64,6 +64,9 @@ If you don't have much familiarity with Kubernetes, we recommend you read the fo
 [**Overview**](overview.md)
 : A brief overview of Kubernetes concepts.
 
+**Cluster**
+: A cluster is the compute resources on top of which your containers are built.
```

Comment:
I intentionally omitted Cluster and Node here, on the theory that those concepts should be defined in the admin guide instead.